cargo-near-build/Cargo.toml

```diff
@@ -24,7 +24,7 @@ zstd = "0.13"
 schemars = "0.8"
 rustc_version = "0.4"
 bon = "2.3.0"
-near-verify-rs = { version = "0.1.0", optional = true }
+near-verify-rs = { version = "0.1.0", git = "https://github.com/dj8yfo/near-verify-rs.git", rev = "bb59fd9fe474697a98fca34bca7c9651ae40e9b4", optional = true }
```

Comment:
todo: replaces with crates.io version after https://github.com/near/near-verify-rs/pull/4 is released

Reply:
handled in https://github.com/near/cargo-near/pull/323/commits/ea4328a4f2af8917aca9bc9125bed508a5c5efe1 after https://github.com/near/near-verify-rs/pull/7